src/components/AssetManagementPanel/AssetManagementForm/FormVariants/ActionSelectionForm/ActionSelectionForm.test.tsx

```diff
@@ -98,12 +102,39 @@ describe("ActionSelectionForm", () => {
         account="0xa61B056dA0084a5f391EC137583073096880C2e3"
         canSurrender={true}
         onConnectToWallet={() => alert("Login to Metamask")}
+        isHolder={true}
       />
     );
 
     expect(container.queryByTestId("SurrenderDropdown")).toBeNull();
   });
 
+  it("should change the state of the application to TransferHolder when we clicked on TransferHolder", async () => {
+    const mockOnSetFormAction = jest.fn();
+
+    const container = render(
+      <ActionSelectionForm
+        onSetFormAction={mockOnSetFormAction}
+        tokenId="0x5d063d51d222c0f5f84fbe18f8e5102859a262f5e1b50148131282d0ebde0066"
+        tokenRegistryAddress="0xdA8DBd2Aaffc995F11314c0040716E791de5aEd2"
+        beneficiary="0xE94E4f16ad40ADc90C29Dc85b42F1213E034947C"
+        holder="0xa61B056dA0084a5f391EC137583073096880C2e3"
+        account="0xa61B056dA0084a5f391EC137583073096880C2e3"
+        canSurrender={true}
+        onConnectToWallet={() => alert("Login to Metamask")}
+        isHolder={true}
+      />
+    );
+
+    await act(async () => {
+      fireEvent.click(container.getByTestId("manageAssetDropdown"));
+    });
+
+    await act(async () => {
+      fireEvent.click(container.getByTestId("transferHolderDropdown"));
```

Comment:
if there is no actions between i think you can put them together in one act bracket.

Reply:
It can't cos the transferHolderDropdown is not rendered yet.